src/apollo/Apollo.ts

```diff
@@ -286,7 +285,7 @@ export default class Apollo implements ApolloInterface, KeyRestoration {
    *  });
    * ```
    *
-   * @param {PrivateKey} privateKey
+   * @param parameters
```

Comment:
Adding `@param` with just the parameter name without any documentation isn't very useful